src/core/core.scale.js

```diff
@@ -730,7 +731,7 @@ module.exports = function(Chart) {
 			if (gridLines.drawBorder) {
 				// Draw the line at the edge of the axis
 				context.lineWidth = helpers.getValueAtIndexOrDefault(gridLines.lineWidth, 0);
-				context.strokeStyle = helpers.getValueAtIndexOrDefault(gridLines.color, 0);
+				context.strokeStyle = helpers.getValueAtIndexOrDefault(gridLines.axisColor, 0);
```

Comment:
zeroLineColor affects only the line at 0, which could be in the middle of the graph(e.g. min: -20, max: 20) or not displayed at all(e.g. min: 50, max: 100).
axisColor is always the border line near ticks

also zeroLineColor affects the other axis(see pic)
<img width="500" alt="axiscolor_zerolinecolor demonstartion" src="https://cloud.githubusercontent.com/assets/19417407/24199325/6495a82a-0f09-11e7-8d75-ec6b8aee6645.png">